app-modules/prospect/database/migrations/2023_07_26_000003_create_prospects_table.php

```diff
@@ -9,7 +9,7 @@ class CreateProspectsTable extends Migration
     public function up(): void
     {
         Schema::create('prospects', function (Blueprint $table) {
-            $table->uuid('id')->primary();
+            $table->string('id')->primary();
```

Comment:
Let's leave a comment here for some context on the decision per the discussion we had when diagnosing the issue.